modules/statics/src/networks.ts

```diff
@@ -389,13 +389,13 @@ class StellarTestnet extends Testnet implements AccountNetwork {
 class Sol extends Mainnet implements AccountNetwork {
   name = 'Sol';
   family = CoinFamily.SOL;
-  explorerUrl = 'https://explorer.solana.com/';
+  explorerUrl = 'https://explorer.solana.com/tx/';
 }
 
 class SolTestnet extends Testnet implements AccountNetwork {
   name = 'SolTestnet';
   family = CoinFamily.SOL;
-  explorerUrl = 'https://explorer.solana.com/?cluster=devnet';
+  explorerUrl = 'https://explorer.solana.com/tx/?cluster=devnet';
```

Comment:
how will this eventually be used in Lumina?
do we have enough info to preserve the query parameter (`?cluster=devnet`) but also append the txid ?

Reply:
In Lumina we are splitting the URL when it has a query param and adding the transaction id in the middle.